modules/introduction/partials/new-features-80.adoc

```diff
@@ -62,6 +62,45 @@ curl --get -u <username:password> \
 [#section-new-feature-data-service]
 === Data Service
 
+[#mb-62777]
+https://jira.issues.couchbase.com/browse/MB-62777[MB-62777] Magma with 128 vBuckets is now the default storage engine::
+In earlier versions of Couchbase Server, if you did not specify which storage engine to use when creating a new bucket, it would use Couchstore.
+Once you upgrade your entire cluster to Couchbase Server Enterprise 8.0, the default storage engine for new buckets is Magma with 128 vBuckets.
+
++
+Magma buckets with 128 vBuckets is a new feature that provides the advantages of the Magma storage engine while reducing the minimum memory quota. 
+Before Couchbase Server 8.0, Magma only supported using 1024 vBuckets and had a minimum memory quota of 1{nbsp}GiB per node.
+Buckets that use Magma with 128 vBuckets have a minimum memory quota of 100{nbsp}MiB per node, which is the same as Couchstore.
+
++
+NOTE: These changes do not affect Couchbase Server Community Edition because it does not support the Magma storage engine.
+
++
+This change in the default storage engine does not affect existing buckets.
+You can still create buckets that use the Couchstore storage engine or the Magma st5orage engine with 1024 vBuckets by explicitly specifying them when you create the bucket.
+
++
+See xref:learn:buckets-memory-and-storage/storage-engines.adoc[] for more information about storage engines.
+
++
+.default behavior change
+[IMPORTANT] 
+====
+The new default storage backend for buckets is a behavior change that may cause issues if you rely on the previous behavior.
+
+Before upgrading to Couchbase Server 8.0, consider the following:
+
+* If you have deployment scripts that create buckets without specifying the storage engine, those scripts  create Magma buckets with 128 vBuckets instead of Couchstore buckets after the upgrade.
+This may affect your deployment if you depend on buckets using the Couchstore storage engine.
+* You cannot use XDCR to replicate Magma buckets using 128 vBuckets with pre-8.0 clusters. 
+XDCR in pre-8.0 clusters only supports replication between buckets that contain the same number of vBuckets.
```

Comment:
This comment is for lines 95 and 96.

I think that this level of detail is good for What's New, but the technical detail is:
You can XDCR **from** a 128 vBucket on 8.0 **to** a pre-8.0 cluster since 8.0 XDCR supports creating a replication between buckets with different numbers of vBuckets.  In XDCR, the source creates the replications.  But, I think that "You cannot use XDCR to replicate Magma buckets using 128 vBuckets with pre-8.0 clusters" is fine for What's New since most people would want to replicate **from** an earlier version **to** the later version or bi-directionally -- so, it gets the most important point across.